client/src/components/browse-recipes/Category.svelte

```diff
@@ -0,0 +1,104 @@
+<script>
+  import { createEventDispatcher, onMount } from 'svelte';
+  import categoriesApi from '@/api/categoriesApi';
+  const dispatch = createEventDispatcher();
+  addEventListener('DOMContentLoaded', event => {
+    let element = document.getElementById('categorySelect');
+
+    element.addEventListener('change', () => {
+      dispatch('categoryToggle', {
+        // @ts-ignore
+        value: element.value,
+        // @ts-ignore
+        text: element.options[element.selectedIndex].text,
+      });
+    });
+  });
+  let categories = [];
+  onMount(async () => {
+    expanded = false;
+    categories = await categoriesApi.categories();
+  });
+
+  let expanded;
+  function expand() {
+    let element = document.querySelector('.categoryContainer');
```

Comment:
this can be done through the style or class attribute on the div itself. Look at my PR for examples